### PR TITLE
PLT-2296 - Adding back button to main create team page

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look. This is to allow users to go back to the other create a team options if he has selected one of them.

### CODE REVIEW

**Comment by hmhealey** [comment]
You've got an ESLint error

image

**Comment by asaadmahmoodspin** [comment]
Fixed.

File: webapp/components/signup_team.jsx [hunk 1]
```diff
@@ -138,6 +138,24 @@ export default class TeamSignUp extends React.Component {
         }
 
         let signupMethod = null;
+        let goBack = (
+            <div className='signup-header'>
+                <a
+                    href='/'
```
Comment (crspeller) [inline webapp/components/signup_team.jsx]: Why `/`? Shouldn't this be `#`?

**Comment by crspeller** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Pull request merged**